setup.cfg

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[metadata]
+name = PennyLane
+version = attr: pennylane._version.__version__
+maintainer = Xanadu Inc.
+maintainer_email = software@xanadu.ai
+license = Apache License 2.0
+description = PennyLane is a Python quantum machine learning library by Xanadu Inc.
+url = https://github.com/XanaduAI/pennylane
+
+project_urls =
+    Tracker = https://github.com/PennyLaneAI/pennylane/issues
+    Documentation = https://pennylane.readthedocs.io/en/stable
+    Source = https://github.com/XanaduAI/pennylane
+
+keywords = 
+    quantum-machine-learning
+    pennylane
+    qml
+
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Operating System :: POSIX
+    Operating System :: MacOS :: MacOS X
+    Operating System :: POSIX :: Linux
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3 :: Only
+    Topic :: Scientific/Engineering :: Physics
+provides = pennylane
+
+[options]
+packages = find:
+install_requires =
```

Comment:
What's the difference between these packages and the system build required packages in `pyproject.toml`?